magicpod-api-client.go

```diff
@@ -197,6 +197,9 @@ func getBatchRunsAction(c *cli.Context) error {
 	count := c.Int("count")
 	maxBatchRunNumber := c.Int("max_batch_run_number")
 	minBatchRunNumber := c.Int("min_batch_run_number")
+	if maxBatchRunNumber < minBatchRunNumber {
+		return cli.NewExitError("--max_batch_run_number value is smaller than --min_batch_run_number value", 1)
+	}
```

Comment:
A better-than-nothing validator. It is OK to omit this one.

Reply:
Adding such check is OK, but now I've got this error when I specified only `min_batch_run_number`.

```
$ ./magicpod-api-client get-batch-runs -min=100
--max_batch_run_number value is smaller than --min_batch_run_number value
```

Reply:
Thank you for pointing that out. Fixed with c6d9503b603bbfca982293c8fb79b3b8b8ee5ec2.